docs/source/io_formats/settings.rst

```diff
@@ -81,6 +81,13 @@ time.
 
     *Default*: 1.0
 
+  :survival_normalization:
+    If this element is set to "true", this will enable the use of survival
+    biasing source normalization, whereby the weight parameters, weight and
+    weight_avg, are multiplied per history by the start weight of said history.
```

Comment:
Based on the current implementation, if we stick with it we probably want to update this language from "history" to "particle".

Reply:
Since this was moved back into the `initialize_history` method, this can be disregarded.